Package.swift

```diff
@@ -17,7 +17,7 @@
 
 import PackageDescription
 
-let googleSignInVersion = "6.1.0"
+let googleSignInVersion = "6.2"
```

Comment:
Can we wait on the version bump?  I usually do this all at once in a separate PR.

Reply:
Sure thing.